templates/OperationResponseNullableBodyEndSnippet.stencil

```diff
@@ -0,0 +1,3 @@
+{% if response.isNullable == true %}
+}
+{% endif %}
```

Comment:
I'm not sure what this is for. NullableBodyEndSnippet? Seems like this should live inline.

Reply:
File is removed.